guides/user/applications/configure.md

```diff
@@ -0,0 +1,144 @@
+
+1. For each event, enter any custom text you want included in the notification.
+
+   For example, for **Any pipeline is complete** you could enter
+   "Congratulations, your pipeline has finished!" Custom notification text is
+   optional.
+
+   ![](/guides/user/applications/event_notification_prefs.png)
+
+1. Click Update
+
+1. Repeat these steps to add more notifications, if you want.
+
+1. Click **Edit** next to any existing notification to change its preferences,
+or click **Remove** to delete it.
+
+## Enable or disable application features
+
+You can disable any of the main features available in your application, if you
+don't need them. Disabling one of these features merely removes it from the
+Deck user interface; no existing data is deleted.
+
+Simply de-select any of the features you don't need:
+
+* pipelines
+* clusters
+* load balancers
+* firewalls
+
+## Add custom instance links
+
+You can include custom links to the instance details panel, providing shortcuts
+to information related to the instance, like logs, health, and so on.
+
+> **Note**: Custom instance links don't currently work for Kubernetes resources.
+
+![](/guides/user/applications/instance_with_links.png)
+
+These links are resolved using the public IP address of the instance, if
+available, or the private IP otherwise. The default port is 80; if you want to
+use a different port, begin the path with `:`.
+
+1. In the **Links** section, click **Add Section**.
+
+1. Provide text for the new **Section heading**.
+
+1. In the **Links** fields, type the text that you want to display, and the path
+to the link target.
+
+   You can reference [these specific instance attributes](/guides/user/instance-links/#available-template-fields/)
+   by wrapping them in curly braces. For example, to include the region where
+   the instance lives, for a GCE or AWS instance, use `{region}`. These are for
+   the path only, not the link display text.
+
+   ![](/guides/user/applications/app_config_add_link.png)
+
+1. Click **Add Link** for each additional link you want displayed in this
+section.
+
+1. Click **Add Section** for any additional sections.
+
+1. Click **Revert** to clear your work in progress.
+
+   This does not remove any saved custom instance links.
+
+1. Click **Save Changes** when you're finished.
+
+Note that you can set default links across all applications, as described
+[here](/guides/user/instance-links/#default-links/).
+
+
+## Refresh caches
+
+Spinnaker keeps a local cache of resources. This section of application
+configuration merely gives you a way to refresh those caches.
+
+## Designate a traffic-guard cluster
+
+A traffic guard is a [cluster](/concepts/clusters/) which you have designated as
+always having at least one active instance.
+
+If anyone tries to delete, disable, or resize a server group, Spinnaker fails
+the operation if it would leave the cluster without any active instances.
+The traffic guard helps ensure this doesn't happen.
```

Comment:
Is the traffic guard the thing that makes sure the operation fails if it would leave the cluster without active instances, or does that happen by default even without a traffic guard? It's not clear to me from the wording of this paragraph.

Reply:
Reworded. Stay tuned for next commit!